engine/src/main/java/org/kakara/engine/item/ItemHandler.java

```diff
@@ -43,6 +43,10 @@ public GameItem getItemWithId(UUID id) {
         return null;
     }
 
+    public List<GameItem> getItems() {
```

Comment:
Why are you adding this back? The items list is empty!

Reply:
I will remove that.